ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/PullQueryPublisher.java

```diff
@@ -124,12 +124,15 @@ public synchronized void subscribe(final Subscriber<Collection<StreamedRow>> sub
           true
       );
 
+      if (pullQueryMetrics.isPresent()) {
```

Comment:
This seems duplicated with line 134-136?

Reply:
Oops! This was a partial refactor I overlooked. Good catch!